tools/accuracy_checker/openvino/tools/accuracy_checker/main.py

```diff
@@ -84,6 +84,7 @@ def main():
                     print_results=True, ignore_results_formatting=args.ignore_result_formatting,
                     ignore_metric_reference=args.ignore_metric_reference
                 )
+
```

Comment:
please revert these changes

Reply:
I think it reverted.

if not args.store_only:
    metrics_results, metrics_meta = evaluator.extract_metrics_results(
        print_results=True, ignore_results_formatting=args.ignore_result_formatting,
        ignore_metric_reference=args.ignore_metric_reference
    )

    if args.csv_result:
        write_csv_result(
            args.csv_result, processing_info, metrics_results, evaluator.dataset_size, metrics_meta
        )